src/tests/playwright/pages/live-suite/scripts/converter.js

```diff
@@ -0,0 +1,47 @@
+import {output} from './console.js';
+
+export default async tests => {
+  const htmls = await Promise.all(tests.map(async t => (await fetch(t)).text()));
+
+  const template = document.querySelector('#row');
+  htmls.forEach((html) => {
+    const dom = new DOMParser().parseFromString(html, 'text/html');
+    const scriptContent = dom.querySelector('script').innerText;
+
+    const tr = template.content.cloneNode(true);
+    const id = dom.title.replace(/[^a-z0-9]/gi, '-').toLowerCase();
+    tr.childNodes[1].id = id;
+    tr.querySelector('slot[name=name]').textContent = dom.title;
+    tr.querySelector('button').addEventListener('click', async () => {
+      document.querySelectorAll('button').forEach(b => b.disabled = true);
+
+      const script = document.createElement('script');
+      script.defer = true;
+      script.type = 'module';
+      script.textContent = scriptContent;
+      document.head.appendChild(script);
+
+      const start = performance.now();
+      // TODO: hacky
+      await new Promise(resolve => setTimeout(resolve, 500));
```

Comment:
Yes. It does look hacky. Can we explain why this is needed for now?

Reply:
If we have to keep it for a while, please add your comment above into the code with `// TODO: `.